main.js

```diff
@@ -15,12 +15,11 @@ chrome.runtime.onInstalled.addListener(function(details) {
     chrome.declarativeContent.onPageChanged.addRules([rule1]);
   });
 });
-  
-  
+
 chrome.webNavigation.onDOMContentLoaded.addListener(function(){
 
   var words = window.localStorage['filterWords'];
-  var words = words.split(',');
+  var words = words.split(', ');
```

Comment:
by including both `,` and `space` the words are effectively trimmed so you don't have leading/trailing spaces